src/containerapp/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 upcoming
 ++++++
+* Update azure cli dependency version >= "2.53.0"
```

Comment:
It is recommended to add the description of the deletion GA commands to the history notes, it will help us trace history in the future

Reply:
Added. Thanks~